### PR TITLE
Fix the DB query in the Versions class

### LINKED ISSUE #3642: Prepared statement fails in Contao\Versions
```An exception occurred while executing a query: SQLSTATE[HY093]: Invalid parameter number: number of bound variables does not match number of tokens```

After pulling in 3cadec7dfd444eefced20da70d5ca21408b3ec82 I get the error message mentioned above after logging into the backend. It originates in the Versions class, when counting versions:

https://github.com/contao/contao/blob/4.x/core-bundle/src/Resources/contao/classes/Versions.php#L655-L656

I logged in as Admin, hence the ` AND userid=?` part was not added to the prepared statement. The user id however is always passed to `execute()`.

### PR DESCRIPTION
**Pull request by leofeyer**
| Q                | A
| -----------------| ---
| Fixed issues     | Fixes #3642
| Docs PR or issue | -

### CODE REVIEW

**Review by Toflar — DISMISSED** [review]
Not sure why the CI stuff is in this PR? But looks good anyway :)

File: core-bundle/src/Resources/contao/classes/Versions.php [hunk 1]
```diff
@@ -651,9 +651,16 @@ public static function addToTemplate(BackendTemplate $objTemplate)
 		$objUser = BackendUser::getInstance();
 		$objDatabase = Database::getInstance();
 
+		$params = array();
+
+		if (!$objUser->isAdmin)
+		{
+			$params[] = $objUser->id;
+		}
+
```
Comment (m-vo) [inline core-bundle/src/Resources/contao/classes/Versions.php]: Less negation and shorter?
```suggestion
		$params = $objUser->isAdmin ? array() : array($objUser->id);
```

**Review by m-vo — DISMISSED** [review]

**Review by bytehead — DISMISSED** [review]

**Review by ausi — DISMISSED** [review]

File: core-bundle/src/Resources/contao/classes/Versions.php (hunk 1 above)
Comment (leofeyer) [reply core-bundle/src/Resources/contao/classes/Versions.php]: Changed in 1f77c2334dddb2a1ca2341f2abb1ac6f49392c48.

**Comment by leofeyer** [comment]
> Not sure why the CI stuff is in this PR? But looks good anyway :)

Removed in 1f77c2334dddb2a1ca2341f2abb1ac6f49392c48.

**Review by ausi — APPROVED** [review]

**Pull request merged**